ADOPTERS.md

```diff
@@ -81,4 +81,5 @@ This list is sorted in the order that organizations were added to it.
 | [Variphy](https://www.variphy.com/) | [info@variphy.com](mailto:info@variphy.com) ([@dmvariphy](https://github.com/dmvariphy) [@nick-variphy](https://github.com/nick-variphy) [@dfalter-variphy](https://github.com/dfalter-variphy) [@zach-variphy](https://github.com/zach-variphy)) | We use Crossplane (via [Upbound Cloud](https://www.upbound.io/)) to manage our development and production infrastructure via GitOps.  Crossplane also allows us to provide custom APIs for production Variphy applications to dynamically manage external resources, such as [Confluent Cloud](https://www.confluent.io/) Kafka topics.  |
 | [OneUptime](https://oneuptime.com) | @simlarsen | Builds production and developer environments that power the OneUptime Platform. |
 | [Xata](https://xata.io) | [@mattfield](https://github.com/mattfield) [@paulaguijarro](https://github.com/paulaguijarro) | Crossplane manages the dev, staging, and production RDS Aurora PostgreSQL clusters for our [Dedicated Clusters](https://xata.io/blog/postgres-dedicated-clusters) offering, along with Flux Kustomizations and other resources that provision cells of internal [Xata](https://xata.io) services. |
-| [AlphaSense](https://www.alpha-sense.com/) | @abhihendre | Engineering teams at [AlphaSense](https://www.alpha-sense.com/) leverage Crossplane APIs, abstracted by a set of Helm charts and Compositions curated by our Platform Teams, to seamlessly provision cloud services across three major clouds, including our production environment.|
\ No newline at end of file
+| [AlphaSense](https://www.alpha-sense.com/) | @abhihendre | Engineering teams at [AlphaSense](https://www.alpha-sense.com/) leverage Crossplane APIs, abstracted by a set of Helm charts and Compositions curated by our Platform Teams, to seamlessly provision cloud services across three major clouds, including our production environment.|
+| [UiPath](https://www.uipath.com/) | [@mjnovice](https://github.com/mjnovice) | Control plane for infrastructure management which powers [AutomationSuite](https://docs.uipath.com/automation-suite/automation-suite/2023.10/installation-guide-eks-aks/automation-suite-on-eksaks-overview) |
```

Comment:
@mjnovice is this also in production environments at UiPath? If so, could you update the text to specifically include something like "in production environments"? That would also helpful to know!

if not, no worries, we can merge as is 😉

Reply:
Hey Jared. We have an experimental product which might end up in production. Will update once that happens.

Reply:
sounds great, looking forward to an update PR when you get there! 😉